### PR TITLE
Fixes runtimes when using supplypod smite on ghosts and fixes supplypod-smite hotspot effects.

### PR DESCRIPTION
**Pull request by MrDoomBringer**
SupplyPod smites weren't creating a hotspot and explosion sound when called on living mobs, and when called on ghosts they wouldn't happen at all. Closes #38626 @Arianya 

:cl: MrDoomBringer
fix: SupplyPod smites work properly once again.
/:cl:

I have a suspicion that this was caused by ~#38374~ an act of god or some shit  , but I'm not certain. Works now though!

### CODE REVIEW

**Comment by SpaceManiac** [comment]
Did you link the right PR? I doubt the bug was caused by PR that hasn't been merged.

**Comment by MrDoomBringer** [comment]
okay so i meant a different PR but its def not even that so idek

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1356,8 +1356,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 					alert("ERROR: Incorrect / improper path given.")
 					return
 			//send the pod
-			target.Stun(10)//takes 0.53 seconds for CentCom pod to land
-			new /obj/effect/DPtarget(get_turf(target), delivery, POD_CENTCOM, target)
+			if(!isobserver(target))
```
Comment (AnturK) [inline code/modules/admin/verbs/randomverbs.dm]: This should be istype check at the start not the other way around.

Comment (MrDoomBringer) [reply code/modules/admin/verbs/randomverbs.dm]: What to you mean "other way around"? Is it better practice to put the isobserver check at the top of that block?

Comment (AnturK) [reply code/modules/admin/verbs/randomverbs.dm]: I mean it shouldn't be checking against a single thing that is excluded, instead check what's allowed and work depending on that.

Comment (MrDoomBringer) [reply code/modules/admin/verbs/randomverbs.dm]: ooh

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -1356,8 +1356,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 					alert("ERROR: Incorrect / improper path given.")
 					return
 			//send the pod
-			target.Stun(10)//takes 0.53 seconds for CentCom pod to land
-			new /obj/effect/DPtarget(get_turf(target), delivery, POD_CENTCOM, target)
+			if(istype(target,mob/living/carbon))
```
Comment (SpaceManiac) [inline code/modules/admin/verbs/randomverbs.dm]: `mob/living` -> `/mob/living`

Comment (MrDoomBringer) [reply code/modules/admin/verbs/randomverbs.dm]: listen i swear i compile my code

**Comment by MrDoomBringer** [comment]
OH MY GOD I DIDDNT COMMIT THE FIX FUCK

**Comment by MrDoomBringer** [comment]
just tested it works

File: code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm [hunk 3]
```diff
@@ -104,7 +104,7 @@
 	light_range = 2
 	var/obj/effect/temp_visual/fallingPod
 
-/obj/effect/DPtarget/Initialize(mapload, var/SO, var/podID, var/target)
+/obj/effect/DPtarget/Initialize(mapload, var/SO, var/podID)
```
Comment (Cyberboss) [inline code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm]: Remove `var/`s

File: code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm [hunk 4]
```diff
@@ -132,6 +132,8 @@
 	else if(podID == POD_CENTCOM)
 		new /obj/structure/closet/supplypod/bluespacepod/centcompod(drop_location(), SO)//CentCom supplypods dont create explosions; instead they directly deal 40 fire damage to people on the turf
 		var/turf/T = get_turf(src)
+		playsound(T, "explosion", 80, 1)
```
Comment (Cyberboss) [inline code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm]: playsound(src, ...

File: code/modules/admin/verbs/randomverbs.dm [hunk 5]
```diff
@@ -1356,8 +1356,9 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 					alert("ERROR: Incorrect / improper path given.")
 					return
 			//send the pod
-			target.Stun(10)//takes 0.53 seconds for CentCom pod to land
-			new /obj/effect/DPtarget(get_turf(target), delivery, POD_CENTCOM, target)
+			if(istype(target, /mob/living/carbon))
```
Comment (Cyberboss) [inline code/modules/admin/verbs/randomverbs.dm]: iscarbon

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm (hunk 3 above)
Comment (MrDoomBringer) [reply code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm]: yikes i should do that from everything huh

File: code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm (hunk 4 above)
Comment (MrDoomBringer) [reply code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm]: look i copypasted from fireballcode

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**